fastapi/middleware/asyncexitstack.py

```diff
@@ -18,7 +18,7 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
                     await self.app(scope, receive, send)
                 except Exception as e:
                     dependency_exception = e
-                    raise e
+                    raise dependency_exception from dependency_exception
```

Comment:
This is more confusing than the 'old' way, the 'from' statement doesn't add anything (nor technically, nor in readability)